stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/input/MessageInputView.kt

```diff
@@ -622,7 +665,17 @@ public class MessageInputView : ConstraintLayout {
 
                     refreshControlsState()
                     handleKeyStroke()
-                    messageInputDebouncer?.submitSuspendable { suggestionListController?.onNewMessageText(messageText) }
+
+                    // Debouncing when clearing the input will cause the suggestion list
+                    // popup to appear briefly after clearing the input in certain cases.
+                    if (messageText == String.EMPTY) {
```

Comment:
could we just say .isEmpty() or .isEmptyOrBlank()

Reply:
Yes, certainly.

Sometimes I do these things because of code consistency. `String.EMPTY` is used elsewhere so I simply continued the chain.

I'll change the other implementations too.